keras/backend/tensorflow_backend.py

```diff
@@ -1926,8 +1927,8 @@ def _fused_normalize_batch_in_training(x, gamma, beta, reduction_axes,
 
     return tf.nn.fused_batch_norm(
         x,
-        gamma,
-        beta,
+        tf.cast(gamma, tf.float32),
```

Comment:
You should only cast a tensor if it isn't already in the right dtype. This applies to all casts here.

Reply:
Changed it, however I believe that tf.cast already handles it:
```python
if isinstance(x,(ops.Tensor, _resource_variable_type)) and base_type == x.dtype:
    return x
```
https://github.com/tensorflow/tensorflow/blob/71e495448c744729c64fcc38b608a029c15828b4/tensorflow/python/ops/math_ops.py#L662